components/list-issues.tsx

```diff
@@ -67,7 +68,8 @@ export default function ListIssues({
   proposer,
   redirect,
   disputableFilter,
-  allNetworks = false
+  allNetworks = false,
+  inView
```

Comment:
This component is used by other pages too, such as `/profile/bounties`, `/profile/pull-request`, `/profile/proposals`, etc. Have you checked that everything is fine with these pages ?

Reply:
yes, they all rendered normally